src/flint/fmpz.jl

```diff
@@ -1304,29 +1304,27 @@ end
 @doc Markdown.doc"""
     fibonacci(x::Int)
 >  Return the $x$-th Fibonacci number $F_x$. We define $F_1 = 1$, $F_2 = 1$ and
-> $F_{i + 1} = F_i + F_{i - 1}$ for all $i > 2$. We require $n \geq 0$. For
-> convenience, we define $F_0 = 0$.
+> $F_{i + 1} = F_i + F_{i - 1}$ for all $i > 2$. The sequence is defined for
+> all integers.
```

Comment:
This begs the question, how is the sequence defined for `i < 0`. I don't know the best way to formulate it, but maybe just replace `for all $i > 2$.` by `for all $i$` ? I can make a PR to update that if you like.

Reply:
Yeah, you can change it to "for all i" if you wish.

Reply:
done: #733